Tasks/UsePythonVersionV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -8,6 +8,8 @@
   "loc.input.help.versionSpec": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://go.microsoft.com/fwlink/?LinkID=2006180)",
   "loc.input.label.allowUnstable": "Allow downloading unstable releases",
   "loc.input.help.allowUnstable": "Allow downloading unstable python versions from [Github Actions python versions registry](https://github.com/actions/python-versions).",
+  "loc.input.label.githubToken": "GitHub token",
+  "loc.input.help.githubToken": "GitHub token that will be used for downloading python versions from [Github Actions python versions registry](https://github.com/actions/python-versions). Not needed if you don't need to download python.",
```

Comment:
Why do we need to use GitHub registry? Are there some other registries we could use?

Reply:
The task is only tuned to work with the GitHub registry. Other registries have a different structure, and thus will require a change in task implementation.